tensorflow/docs_src/programmers_guide/datasets.md

```diff
@@ -44,7 +44,7 @@ To start an input pipeline, you must define a *source*. For example, to
 construct a `Dataset` from some tensors in memory, you can use
 `tf.data.Dataset.from_tensors()` or
 `tf.data.Dataset.from_tensor_slices()`. Alternatively, if your input
-data are on disk in the recommend TFRecord format, you can construct a
```

Comment:
"data are on disk" is correct; "the recommend TFRecord format" is obviously wrong. Please revert the first change and keep the second one.

Reply:
done